File: Tecnico.py
from Pessoa import Pessoa

class Tecnico(Pessoa):

    def __init__(self, nome, time, salario):
        super().__init__(nome, time)
        self.__salario = salario
        print("Técnico Criado")

    def get_salario (self):
        return self.__salario

    def set_salario (self, salario):
        self.__salario = salario

    def calcularSalarioAnual (self):
        return self.__salario*12

    def apresentacao(self):
        print(self._nome, "é o técnico do time", self._time)

File: ProjetoJava/src/main/java/source/projetojava/Tecnico.java
package source.projetojava;

/**
 *
 * @author alejb
 */
public class Tecnico extends Pessoa {

  float salario;
  
  public Tecnico() {

    System.out.println("Técnico criado");
        
  }
  
  public Tecnico(String nome, String time, float salario) {
  
    super (nome, time);
    this.salario = salario;
    System.out.println("Técnico criado");
      
  }

  @Override
  public void apresentacao () {
      
    System.out.println(this.nome + " é o técnico do time " + this.time + ".");
    
  }
  
  public void setSalario(float salario) {
    this.salario = salario;
  }
  
  public float getSalario() {
    return this.salario;
  }
  
  public float calcularSalarioAnual () {  
    return (float) (this.salario*12.00);
   }
  
}
